sqlx-core/Cargo.toml

```diff
@@ -18,6 +18,8 @@ any = []
 
 json = ["serde", "serde_json"]
 
+full_text_search = []
```

Comment:
Why add this feature to `sqlx-core`?

Reply:
@abonander Thanks for pointing that out. It makes sense. Will remove.

Reply:
@abonander I've updated this PR to remove the feature flag. Please review again.